step-maven-plugin/step-ee-maven-plugin/src/main/java/step/plugins/maven/RunPackagedExecutionBundleMojoEE.java

```diff
@@ -0,0 +1,25 @@
+package step.plugins.maven;
+
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import step.client.credentials.ControllerCredentials;
+
+@Mojo(name = "run-packaged-execution-bundle-ee")
+public class RunPackagedExecutionBundleMojoEE extends AbstractRunPackagedExecutionBundleMojo {
```

Comment:
The tenant selection is missing here

Reply:
@jeromecomte do we need to apply the same tenant selection logic as in UploadKeywordsPackageMojoEE, am I right?